build.dist.xml

```diff
@@ -9,11 +9,13 @@
     <!-- Make these commands available by default. -->
     <import file="vendor/palantirnet/the-build/tasks/drupal.xml" />
     <import file="vendor/palantirnet/the-build/tasks/acquia.xml" />
+    <import file="vendor/palantirnet/the-build/tasks/styleguide.xml" />
 
 
     <!-- Default target: build -->
     <target name="build" description="Build the application.">
         <phingcall target="drupal-build" />
+        <phingcall target="styleguide" />
```

Comment:
I'm testing this on a project that uses the `copy` instead of `symlink` artifact mode.  Does it make sense to move the `<phingcall>` above the build, so the styleguide assets are updated before being copied into Drupal's theme?

Reply:
@byrond Yeah that makes sense to have a copy/updated before being copied over into Drupal's theme. Maybe with an output of what is copied over to help users debug - if there is a need to troubleshoot/debug changes.

Reply:
Thanks! I've made this change, and I've added an example of how to include theme assets using tools provided by the-build.